src/shared/bus-unit-util.c

```diff
@@ -712,8 +712,7 @@ int bus_append_unit_property_assignment(sd_bus_message *m, const char *assignmen
                 if (r < 0)
                         return bus_log_create_error(r);
 
-                if (cpuset)
-                        sd_bus_message_append_array(m, 'y', cpuset, CPU_ALLOC_SIZE(ncpus));
+                sd_bus_message_append_array(m, 'y', cpuset, CPU_ALLOC_SIZE(ncpus));
```

Comment:
I'd love to see an explanation in the commit message what the user-facing effect of this is. E.g. "Allows the user to reset CPUAffinity= back to an empty list" or whatever.

Reply:
Thank you. updated